app/views/purchases/index.html.erb

```diff
@@ -0,0 +1,22 @@
+<div class="navigation">
+  <ul class="flex justify-between blue-btn items-center">
+    <li class="w-10"><%= link_to image_tag('backarrow.png', alt: 'Back'), category_path(params[:category_id]), class: 'nav-link' %></li>
+    <h2>Purchases</h2>
+    <li><%= link_to 'Sign Out', destroy_user_session_path, data: { turbo_method: :delete }, class: "nav-link" %></li>
+  </ul>
+</div>
+
+<div class="purchases-container">
+  <div>    
+    <% @purchases.each do |purchase| %>
+      <div class="purchase-item card-item">
+        <%= link_to purchase.name, category_purchase_path(@category, purchase) %>
+        $ <%= purchase.amount %>
+      </div>
+    <% end %>
+  </div>
+  <div class="flex links flex-col text-center">
+      <%= link_to "Add a New Purchase", new_category_purchase_path(@category), class: "green-btn" %>
+      <%= link_to "Categories", categories_path , class: "categories-btn" %>
+  </div>
```

Comment:
- [ ] For the `Purchases#Index` page, the **_ADD A NEW PURCHASE_** button should be positioned at the bottom of the page based on the Behance design. Currently, the button is not designed that way. Kindly redesign the button so that your application looks identical to that of the Behance design. 😎 💻 

| Your Design  | Behance Design |
| ------------- | ------------- |
| ![image](https://github.com/Katended/budget-trracker/assets/77106962/6cb8c681-d9b6-444f-a03e-60bdec7aa229) | ![image](https://github.com/yegonkimutai/budget-app/assets/77106962/7514c6b7-46db-41fe-ad8d-7a4902203586)  |

---

Please take note of the padding for the button and the inner white border line. Please try your best to make the button look identical to that of the Behance design. 👍 🖱️

Reply:
- [ ] Just as the previous reviewer mentioned and added a screenshot, the button is still not at the bottom just as the given design said, kindly implement this, just as you have already done in the categories index page